Applications: Use thousands separator for numbers shown in status bar

tcl3 opened the pull request:
TextEditor, HackStudio and SQLStudio now print the current line and column number, as well as the number of currently selected words, with thousands separators.

TextEditor also uses thousands seperators for the current word and character count.

NB: While testing I found that the line and column count can get cut off when the line and column number have more than 7 digits combined. This PR does not change that behavior.

Example:
![statusbar_thousands_separators](https://user-images.githubusercontent.com/2817754/231230946-cc77ffd6-0a0f-4635-9006-30a2f3fec3d3.png)

caoimhebyrne reviewed (approved):
Looks good to me, a subtle change but helps with readability, very nice :)

nico commented:
Should maybe drop the `,` in `Ln X, Col Y` so that it just reads `Ln X Col Y` now that X can contain a `,`.

tcl3 commented:
> Should maybe drop the `,` in `Ln X, Col Y` so that it just reads `Ln X Col Y` now that X can contain a `,`.

That makes sense. I've replaced the comma with an extra space; a single space makes it less readable IMO.

I've updated the screenshot above to show what it looks like now.
